lib/taskana-test-api/src/main/java/pro/taskana/testapi/OracleSchemaHelper.java

```diff
@@ -0,0 +1,56 @@
+package pro.taskana.testapi;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import javax.sql.DataSource;
+
+import pro.taskana.common.api.exceptions.SystemException;
+
+public final class OracleSchemaHelper {
+
+  private static final String DEFAULT_PASSWORD = "testPassword";
+
+  private OracleSchemaHelper() {
+    // hide implicitpublic one
+  }
+
+  public static void initOracleSchema(DataSource dataSource, String schemaName)
+      throws SystemException {
+    try (Connection connection = dataSource.getConnection()) {
+      Connection conn =
+          DriverManager.getConnection(
+              connection.getMetaData().getURL(), "SYSTEM", DEFAULT_PASSWORD);
```

Comment:
just out of curiosity. Don't we have to put this connection into the try-with-resources statement above? Otherwise we are not closing it.

Reply:
Done